app/javascript/stylesheets/gobierto-data.scss

```diff
@@ -1324,4 +1403,41 @@
     padding: 0 1rem 1rem 1rem;
   }
 
+  &-description-details {
+    margin: .5rem 0;
+  }
+
+  &-description-details-orders:first-of-type,
+  &-description-details-columns {
+    margin-top: 2rem;
+  }
+
+  &-description-summary-title {
+    cursor: pointer;
+    &::marker {
+      color: var(--color-base);
+    }
+  }
+  &-description-summary-grid,
+  &-description-summary-grid-two-columns {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    padding: 0 .25rem;
+  }
+  &-description-summary-grid-two-columns {
+    grid-template-columns: repeat(2, 1fr);
+  }
+  &-description-summary-grid-element {
+    display: grid;
+    grid-template-columns: 75% 1fr;
+    margin-bottom: .125rem;
+  }
+  &-description-summary-grid-element-link {
+    color: var(--color-base);
+  }
+  &-description-summary-grid-element-size {
+    color: #9B9B9B;
+  }
```

Comment:
This file is massive. You should think about how can it be chunked, by components or similar

Reply:
I've created two new `SCSS`, one for the summary tab and one for the editor tab.